src/EFCore.PG/Metadata/NpgsqlIndexAnnotations.cs

```diff
@@ -51,8 +52,24 @@ public string Method
             set => SetMethod(value);
         }
 
+        /// <summary>
+        /// The PostgreSQL index operators to be used.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/static/indexes-opclass.html
+        /// </remarks>
+        public string[] Operators
```

Comment:
Let's use `IReadOnlyList<string>` here too. Everything else is fine, so I think, @austindrenski could merge it without an additional approval.

Reply:
The setter is for the collection, not the elements, and `IReadOnlyList<string>` is only read-only for the elements... Am I missing something?

Reply:
This is supposed to be mutable, right?

Reply:
The extension method on `IMutableIndex` returns `NpgsqlIndexAnnotations`, which is supposed to be mutable. The `IIndex` overload returns `INpgsqlIndexAnnotations`, which is immutable. Am I missing something?

https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/2a71c936cb040f0111b1d0305e8e409abbb6593e/src/EFCore.PG/Extensions/NpgsqlMetadataExtensions.cs#L47-L51

Reply:
@khellang I think you're right on this. Thanks for pointing this out.